src/v2/guide/list.md

```diff
@@ -489,11 +489,14 @@ Here's a complete example of a simple todo list:
 
 ``` html
 <div id="todo-list-example">
-  <input
-    v-model="newTodoText"
-    v-on:keyup.enter="addNewTodo"
-    placeholder="Add a todo"
-  >
+  <form v-on:submit.prevent="addNewTodo">
+    <input
+      aria-label="Write a new todo item"
```

Comment:
After speaking with an accessibility expert, my understanding is that if we're going to include a placeholder, it would be obscured by this aria-label. It would probably be better to use a label element: `<label for="new-todo">Add a todo</label>` and add an id on the input, that way the placeholder will still be accessible.

Reply:
I agree with using a label. I also think a label element would be more familiar (and therefor less distracting) for readers than an aria-label attribute.

For the sake of my own understanding, would you know how the placeholder is obscured by aria-label?